packages/core/strapi/lib/services/entity-service/types/index.d.ts

```diff
@@ -1 +1,6 @@
+// TODO: Move params to @strapi/utils (related to convert-query-params)
```

Comment:
is this a "TODO V5" or do we have intentions to actually do it before then?

Reply:
It was more of a TODO when we will move the overall architecture (where to export common types from, etc...)
So probably, between end of V4 & V5